diff --git a/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt b/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt
index fbb6836..e75b6b2 100644
--- a/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt
+++ b/node_modules/react-native-document-scanner-plugin/android/src/main/java/com/documentscanner/DocumentScannerModule.kt
@@ -51,7 +51,7 @@ class DocumentScannerModule(reactContext: ReactApplicationContext) :
 
     val documentScannerOptionsBuilder = GmsDocumentScannerOptions.Builder()
       .setResultFormats(GmsDocumentScannerOptions.RESULT_FORMAT_JPEG)
-      .setScannerMode(GmsDocumentScannerOptions.SCANNER_MODE_FULL)
+      .setScannerMode(GmsDocumentScannerOptions.SCANNER_MODE_BASE)
 
     if (options.hasKey("maxNumDocuments")) {
       documentScannerOptionsBuilder.setPageLimit(
